prow/gerrit/adapter/adapter.go

```diff
@@ -394,9 +394,8 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 	case client.New:
 		presubmits, err := cache.GetPresubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
 		if err != nil {
-			//TODO(mpherman): Return Error once we know this usually works
 			logger.WithError(err).Warn("Failed to get cached InRepoConfig for Presubmits.")
```

Comment:
Not sure whether this is still necessary since the error is returned, same above